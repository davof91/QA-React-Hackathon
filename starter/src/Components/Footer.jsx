import React from 'react';

const Footer = () => {
    return (
        <>
            <footer class="mt-auto py-3 text- center container">
                Â©QA Ltd 2019-
            </footer>
        </>
    );
}

export default Footer;