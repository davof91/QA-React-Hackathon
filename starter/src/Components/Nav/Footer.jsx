import React from 'react';

const Footer = () => {
    return (
        <>
            <hr/>
            <footer className="container text-muted" id="footer-txt">
                Â© QA Cinemas 2021
            </footer>
        </>
    );
}

export default Footer;