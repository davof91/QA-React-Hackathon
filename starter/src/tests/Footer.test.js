import React from 'react';
import { create } from 'react-test-renderer';
import Footer from '../Components/Footer';
test('footer matches snapshot', () => {
  const footer = create(<Footer />);
  expect(footer.toJSON()).toMatchSnapshot();
});