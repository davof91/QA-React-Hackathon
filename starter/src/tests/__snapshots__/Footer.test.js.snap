// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer matches snapshot 1`] = `
Array [
  <hr />,
  <footer
    className="container text-muted"
    id="footer-txt"
  >
    Â© QA Cinemas 2021
  </footer>,
]
`;
